datafusion/core/tests/sql/aggregates.rs

```diff
@@ -771,31 +295,6 @@ async fn count_distinct_integers_aggregated_multiple_partitions() -> Result<()>
     Ok(())
 }
 
-#[tokio::test]
-async fn aggregate_with_alias() -> Result<()> {
```

Comment:
this looks like a dataframe test -- I'll port it over to https://github.com/apache/arrow-datafusion/blob/main/datafusion/core/tests/dataframe/mod.rs as a follow on PR